env0/provider.go

```diff
@@ -174,7 +174,7 @@ func Provider(version string) plugin.ProviderFunc {
 }
 
 func createRestyClient(ctx context.Context) *resty.Client {
-	isIntegrationTest := false
+	var isIntegrationTest bool
```

Comment:
?

Reply:
by default a bool var is false. linter complained about this.